props/dual_prop.h

```diff
@@ -112,24 +120,24 @@ class SaberBlasterProp : public virtual Saber, public virtual Blaster {
   }
   const char* name() override { return "DualProp"; }
   bool Event(enum BUTTON button, EVENT event) override {
+
+#ifdef BLADE_DETECT_PIN
     if (button == BUTTON_BLADE_DETECT) {
       if (event == EVENT_LATCH_ON) {
-	Saber::Off();
-	Saber::blade_detected_ = true;
-	Saber::FindBladeAgain();
-	SaberBase::DoBladeDetect(true);
-	current_modifiers = 0;
-	return true;
+        Saber::blade_detected_  = true;
+        SaberBase::DoBladeDetect(true);
       } else  if (event == EVENT_LATCH_OFF) {
-	Saber::Off();
-	Saber::blade_detected_ = false;
-	Saber::FindBladeAgain();
-	SaberBase::DoBladeDetect(false);
-	current_modifiers = 0;
-	return true;
+        Saber::blade_detected_  = false;
+        SaberBase::DoBladeDetect(false);
```

Comment:
I think DoBladeDetect need to be done after FindBladeAgain()

Reply:
indeed.